corehq/apps/app_manager/xpath.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import unicode_literals
```

Comment:
Thanks! Responded [on that thread](https://github.com/dimagi/commcare-hq/commit/57befb3ca35d318df7ac8caab7e2408c4235aeec#commitcomment-17209244). Great points, and I'll drop that commit.